src/states/elder/mod.rs

```diff
@@ -1312,12 +1273,18 @@ impl Base for Elder {
             ParsecResponse(version, par_response) => {
                 return self.handle_parsec_response(version, par_response, pub_id, outbox);
             }
-            Relocate(details) => {
-                return Ok(self.handle_relocate(details));
-            }
             BootstrapResponse(_) => {
                 debug!("{} Unhandled direct message: {:?}", self, msg);
             }
+            msg @ Relocate(_) => {
+                debug!(
+                    "{} Unhandled Elder direct message from {}, adding to backlog: {:?}",
```

Comment:
Nitpick: Just say "Unhandled direct message...". No need to specify it's elder as the message already contain that (via the `Display` impl of `Elder`).

Reply:
I was thinking it would be easier to search for in logs this way because other wise we need to use regexp or a specific elder to follow, so not ideal.

Reply:
> need to use regexp

Is that a problem? Don't you use regexp (grep, etc...) already when analyzing logs?

Reply:
Well, I use it when I have too, but it is a lot easier when things don't need to (in particular with very large log files where regex find slow things down quite a bit), can just select a piece of text and see all places, can also select it directly from the code to find it.

Reply:
> where regex find slow things down quite a bit

Use [ripgrep](https://github.com/BurntSushi/ripgrep) :D

But this is a minor thing so if it makes life easier for you then I'm OK with it.